src/foundry/client/apps/av/av-config.d.mts

```diff
@@ -48,6 +62,30 @@ declare global {
      */
     protected _onTurnTypeChanged(event: JQuery.ChangeEvent): void;
 
-    protected override _updateObject(event: Event, formData?: object): Promise<unknown>;
+    protected override _updateObject(event: Event, formData?: object): Promise<void>;
```

Comment:
What's the better practice here, `Promise<void>` or `Promise<unknown>`, Foundry documents it as void (and from the code it never returns) so to me this seems like the better principle?

Reply:
Void does seem more correct, given that the return isn't used.